fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/rules/PushRequestedOrderingThroughSortRule.java

```diff
@@ -60,27 +61,28 @@ public PushRequestedOrderingThroughSortRule() {
 
     @Override
     public void onMatch(@Nonnull final CascadesRuleCall call) {
-        final var bindings = call.getBindings();
+        final PlannerBindings bindings = call.getBindings();
```

Comment:
`var` vs explicit type. Did you find the `var`s to be hard to read here? Genuinely curious about when we should adopt which style.

Reply:
I can't bring myself to put generic bounds on the RHS ctor call, so there need to be explicit types for those at least. Which leads to mixed style, which I guess we have to agree is mostly okay.

When refactoring and changing the types of things that interact with generics, like `collection.contains`, I found seeing the explicit types helpful, which is why I changed more than the minimum number here. Although the IDE mostly isn't confused, probably.

In brand new code, I think I only _really_ prefer `var` as a standin for type names that are (1) obvious (I know that's subjective) and (2) very long (ditto).